client/src/app/shared/user/auth.effect.ts

```diff
@@ -4,32 +4,32 @@ import { Actions, ofType, Effect } from '@ngrx/effects';
 import { of } from 'rxjs';
 import { exhaustMap, map, catchError, tap } from 'rxjs/operators';
 import { HttpService } from '../../core/http/http.service';
-import { AuthActionTypes, AuthSuccessAction, AuthFailureAction } from './auth.action';
+import { AuthActionTypes, auth, authSuccess, authFailure } from './auth.action';
 import { User } from './user';
 
 @Injectable()
 export class AuthEffect {
   @Effect()
   auth$ = this.actions$.pipe(
-    ofType(AuthActionTypes.AUTH),
+    ofType(auth.type),
     exhaustMap(() => {
       return this.httpService.get<User>('/me/session').pipe(
-        map(user => new AuthSuccessAction({ user })),
-        catchError(error => of(new AuthFailureAction({ error: error.error })))
+        map(user => authSuccess({ user })),
+        catchError(error => of(authFailure({ error: error.error })))
```

Comment:
__[TSLint]__ of is deprecated: use {@link scheduled} instead `scheduled([a, b, c], scheduler)` (deprecation) ([link](https://sider.review/gh/repos/72339652/pulls/67#issue-4029941))

<sub>posted by Sider</sub>